rag/advanced_rag/knowlege_compile/wiki_incremental.py

```
@@ -0,0 +1,1619 @@
+    from common.doc_store.doc_store_base import OrderByExpr
+
+    query_texts = [_entity_to_query_text(e) for e in affected_entities]
+    embeddings, _ = await thread_pool_exec(embd_mdl.encode, query_texts)
+
+    index = search.index_name(tenant_id)
+    condition = {"compile_kwd": [WIKI_PAGE_COMPILE_KWD]}
+
+    assignments: dict[str, list[dict]] = {}
+    orphans: list[dict] = []
+
+    for entity, vec in zip(affected_entities, embeddings):
```

Ruff (0.16.0)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)